release-notes-1.1.1.md

```diff
@@ -1,26 +1,75 @@
-# Overview to Ballerina 1.1.1
+# Overview of Ballerina 1.1.1
 
-Ballerina 1.1.1 is a patch release iteration of its previous 1.1.0 release. Please refer to the Github repository for a list of [bug fixes](https://github.com/ballerina-platform/ballerina-lang/issues?utf8=✓&q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+1.1.1%22+is%3Aclosed+)
-and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?utf8=✓&q=is%3Aissue+milestone%3A%22Ballerina+1.1.1%22+is%3Aclosed+label%3AType%2FImprovement+).
+Ballerina 1.1.1 is a patch release iteration of its previous 1.1.0 release, which introduces the below functionalities. Also, this release addresses a few [bug fixes](https://github.com/ballerina-platform/ballerina-lang/issues?utf8=✓&q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+1.1.1%22+is%3Aclosed+)
+and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?utf8=✓&q=is%3Aissue+milestone%3A%22Ballerina+1.1.1%22+is%3Aclosed+label%3AType%2FImprovement+). 
 
-## How to update
+You can use the inbuilt updating capability (introduced in Ballerina 1.1.0) to update to JBallerina 1.1.1 by executing the respective command from the below list.
 
-Ballerina 1.1.0 has introduced an inbuilt updating capability into it. Therefore, if you have installed Ballerina 1.1.0 and if you are using jballerina-1.1.0, you can simply run the following command to update Ballerina.
+<table>
+<tr border=1>
+<th> Installed version </th>
+<th> Current active version </th>
+<th> Command </th>
+</tr>
+<tr border=1>
+<td>  jBallerina-1.1.0 </td>
+<td>  jBallerina-1.1.0 </td>
+<td> `ballerina dist update` </td>
```

Comment:
It seems the backtick is not properly highlighting this text.

Reply:
Fixed.